src/main/kotlin/io/kotlintest/properties/Gen.kt

```diff
@@ -1,67 +1,70 @@
 package io.kotlintest.properties
 
-import com.sksamuel.koors.Random
 import java.lang.reflect.ParameterizedType
 import java.lang.reflect.Type
 import java.lang.reflect.WildcardType
+import java.util.*
+
+/** A shared random number generator. */
+private val RANDOM = Random()
 
 interface Gen<T> {
   fun generate(): T
 
   companion object {
 
     fun choose(min: Int, max: Int): Gen<Int> = object : Gen<Int> {
-      override fun generate(): Int = Random.default.nextInt((max.toLong() - min.toLong()).toInt()) + min
+      override fun generate(): Int = RANDOM.nextInt((max.toLong() - min.toLong()).toInt()) + min
     }
 
     fun choose(min: Long, max: Long): Gen<Long> = object : Gen<Long> {
       override fun generate(): Long {
-        var rand = (Random.default.nextLong() % (max - min));
-        if(rand < 0) {
-          rand += max - min;
+        var rand = (RANDOM.nextLong() % (max - min))
+        if (rand < 0) {
+          rand += max - min
         }
-        return rand + min;
+        return rand + min
       }
     }
 
     fun <T> oneOf(values: List<T>): Gen<T> = object : Gen<T> {
-      override fun generate(): T = Random.default.shuffle(values).first()
+      override fun generate(): T = values[RANDOM.nextInt(values.size - 1)]
```

Comment:
Using `shuffle` seemed a bit redundant for generating a single item, so I've replaced it with a simpler version.